app/components/article-featured-video.js

```diff
@@ -24,6 +24,46 @@ export default Component.extend(
 			this._super(...arguments);
 
 			this.initVideoPlayer();
+			this.initOnScrollBehaviour();
+		},
+
+		/**
+		 * Manages video transformation on user's scroll action
+		 *
+		 * @returns {void}
+		 */
+		initOnScrollBehaviour() {
+			let $video = this.$('.video-container'),
+				$siteHead = $('.site-head'),
+				videoBottomPosition = $video.offset().top + $video.height(),
+				showVideoOnScroll = true;
+
+			$(window).on('scroll', {player: this.player, $window: this.$(window)}, function ({data: {player, $window}}) {
```

Comment:
Do we need to pass those parameters on every callback? Even when they're not use because throttle decides is not required?